news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+- You may collaborate if you like, but only the submitter of each entry will receive credit for judging and prizes.
+- Maps submitted to the contest must not be uploaded to the BSS until the contest has concluded and the winners have been announced.
+- **Send your finished beatmap as .osu-file to XXX before the deadline expired. Any submissions received after the deadline expired will not be accepted.**
+
+### Judging Criteria
+
+We have also slightly reworked the judging criteria to provide a more reasonable judging base. Down below you can see what your map will be judged for:
+
+- Musical Relevance (10P) - How well does the entry follow the music with its patterns, its structure and overall concept?
+- Creativity (10P) - How unique and/or innovative is the entry in its use of concepts and overall design?
+- Technique (10P) - How well does the entry utilize the editor and its tools in its creation? 
+- Impression (10P) - How does the judge view the map in its entirety?
+- Hitsounding (5P) - How well does the entry utilize hitsounding to emphasize and follow the music?
+
+**Speaking of Judging: We are still looking for capable judges! Are you interested? Feel free to drop a PM to Irreversible with your motivation, knowledge and possible references, where you've already judged. As a compensation, you will receive 2 months of supporter. Please be aware that we can only accept people who do have enough ressources (knowledge, time, reliability) to successfully do this.**
+
+### Prizes
+
+After the entries have been judged and the results finalized, each of the top three maps for each track will receive the following prizes:
+
+- **1st place**: 6 months of osu!supporter, <Olympiad> Winner Badge
+- **2nd place**: 4 months of osu!supporter, <Olympiad> Honorable Mention Badge
+- **3rd place**: 2 months of osu!supporter, <Olympiad> Honorable Mention Badge
+
+Now, you may be wondering why there is a lack of *Elite Mapper* forum title listed in the prizes. After much deliberation, we felt that granting the *Elite Mapper* title to each song’s top placing entrant would not be doing the prestige of the title enough justice. 
+
+Instead, the *Elite Mapper* title for this contest will be given to the winner of a second contest following this one, which the winning mapper from each track will compete in! 
+
+The winner of this additional contest will be given the honor of becoming osu!’s newest **Elite Mapper** (or special incrementation should they already possess the title)!
+_Note: The Title is subject of change._
+
+The contest will remain open for **30 days** from the date of this post. Download the pre-timed .osz files below:
+
+[Song 1: Kuba Ohm - Beautiful Uncertainty (Redux)](http://puu.sh/y77Cs/400fb0ae01.osz)
+
+[Song 2: OISHII - HONEY SNOWFLAKES](http://puu.sh/y77Ga/acc4864a52.osz)
+
+[Song 3: cySmix - Escapism](http://puu.sh/y7Vr4/3b1eb163c2.osz)
+
+But enough explanations - get out and have fun mapping, we expect some awesome entries!
```

Comment:
add the name of the one posting this in a format like `—Username` in a separate line underneath the ending sentence to keep coherent with all other newsposts